README.md

```diff
@@ -72,11 +72,13 @@ First, you're going to need to copy the `.env.example` to `.env`. The current op
 | `PHONE_CARRIER` | [Supported carriers](#supported-carriers) for SMS; optional, email configuration required |
 | `RATE_LIMIT_TIMEOUT` | Rate limit timeout for each full store cycle; optional, default: `5000` |
 | `SLACK_CHANNEL` | Slack channel for posting (e.g., `update`); optional |
-| `SLACK_TOKEN` | Slack API token; optional
+| `SLACK_TOKEN` | Slack API token; optional |
 | `STORES` | [Supported stores](#supported-stores) you want to be scraped; optional, default: `nvidia` |
 | `OPEN_BROWSER` | Toggle for whether or not the browser should open when item is found, default: `true` |
 | `PLAY_SOUND` | Play this sound notification if a card is found.; optional |
 | `SCREENSHOT` | Capture screenshot of page on successful hit; optional, default `true` |
+| `TELEGRAM_BOT_TOKEN` | Telegram bot token; optional |
+| `TELEGRAM_CHAT_ID` | Telegram chat ID; optional |
```

Comment:
Perhaps add the `SHOW_ONLY_BRANDS` description?

Reply:
Whoops, forgot that one.